lib/FusionInventory/Agent/Task/Inventory/Virtualization/Qemu.pm

```diff
@@ -33,6 +33,8 @@ sub _parseProcessList {
             $values->{name} = $1 if !$values->{name};
         } elsif ($option =~ m/^name (\S+)/) {
             $values->{name} = $1;
+        } elsif ($option =~ m/^\/usr\/(s?)bin\/(\S+)/) {
+            $values->{vmtype} = $2;
```

Comment:
Indeed I would better see to set vmtype to "kvm" or "qemu" with the following code:
```suggestion
            $values->{vmtype} = $2 =~ /kvm/ ? "kvm" : "qemu";
```
The process is matching: `/(qemu|kvm|qemu-kvm|qemu-system\S+) .*\S/x` on l.72 so just reporting the command as type can lead us to a bunch of new and not useful vmtype as qemu has changed it's command usage other the time.
"qemu" vs "kvm" means more with and without CPU emulation.